src/Raygun.Blazor/Models/StackTraceDetails.cs

```diff
@@ -87,10 +87,22 @@ internal StackTraceDetails(StackFrame frame)
             ColumnNumber = frame.GetFileColumnNumber();
             FileName = frame.GetFileName();
             ILOffset = frame.GetILOffset();
-            //ImageSignature = frame.ImageSignature;
             LineNumber = frame.GetFileLineNumber();
             MethodName = names.MethodName;
             MethodToken = frame.GetMethod()?.MetadataToken;
+
+            // Attempt to get the image signature from the debug information
```

Comment:
You should test whether this works on MAUI Blazor Hybrid apps specifically in Android. We had to do quite a bit of work to get the image there. 

Check out this PR for some more context https://github.com/MindscapeHQ/raygun4maui/pull/20

Reply:
Thanks for the hint! When I did the testing on Android, I was able to obtain the PE Debug data from the frames. The only platform where it didn't work was WebAssembly Blazor apps. I will check again once when we have the solution working end to end on Blazor Server.